extras/http2/src/javascripts/app.js

```diff
@@ -0,0 +1,5 @@
+// es6-promise/auto needed to fulfil promises for the `import()` statement in modules/index.js
+require('es6-promise/auto')
```

Comment:
Any reason why this can't be `import 'es6-promise/auto'`?

Reply:
this specific module allows older browsers to use the `Promise` feature on imports with a polyfill: https://github.com/stefanpenner/es6-promise, but i don't think in this particular instance a promise is needed so it can be imported. meta.